lib/conversion.js

```diff
@@ -2,21 +2,36 @@ var extend = require('extend');
 var streamMixins = require('./mixins/_stream');
 var readableMixins = require('./mixins/_readable');
 var writableMixins = require('./mixins/_writable');
+var native = require('./native');
 
-function isRequestStream(stream) {
-	return !!(stream.pipe && !stream.read && stream.headers);
+function isStreams1(stream) {
+	return !!(!(stream instanceof native.Readable) &&
+		!(stream instanceof native.Writable) &&
+		!(stream instanceof native.Duplex) &&
+		stream instanceof native.Stream &&
```

Comment:
This won't consistently work if native is configured to use readable-stream.  We should (essentially) never use instanceof with stream classes because it won't be reliable.  Instead we should use duck typing.